_posts/2020-08-18-network-upgrade-october-2020.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Monero will experience a network upgrade on 17th October 2020"
+summary: Announcement of the upcoming network upgrade (17th October)
+tags: [community, protocol upgrade, core, crypto]
+author: ErCiccione
+---
+
+The next network upgrade will take place on **17th October 2020**. Network upgrades are technically hard forks and require all the partecipants to the Monero network to upgrade their software (@wallet and @daemon). The new software will be released one month before the network upgrade: **17th September**.
+
+This network upgrade will introduce ground changing features like the [new ring signature construction CLSAG]({{ site.baseurl_root }}/2020/07/31/clsag-audit.html), which will drastically **reduce the transaction size by ~25%** and **improve verification performance by ~20%**. Basically, everything will be lighter and faster. The complete list of changes and improvements will be released along with the new binaries.
+
+The logs of the meeting where the dates of the network upgrade were decided [are available on this website]({{ site.baseurl_root }}/2020/07/19/monero-dev-meeting.html).
+
+### Common questions
+
+**Q:** Hard fork? does this mean there will be a split and a new coin will be created?  
+**A:** *No. Hard forks in Monero are made to improve the base protocol, they are non-contentious. There will be no split and no new coin will be created.*
+
+**Q:** What do I have to do to be prepared for this upgrade?  
+**A:** *A new release will be announced before the network upgrade (Around 17th September). You will only have to be using the updated software by the time the network upgrade will occur (17th October). To the final user, it will be like .a simple software update.*
```

Comment:
```suggestion
**A:** *A new release will be announced before the network upgrade (Around 17th September). You will only have to be using the updated software by the time the network upgrade occurs (17th October). To the final user, it will be like a simple software update.*
```

Reply:
Edited anyway. Should be the same.